src/stories/Quiz/Component/ComponentStories.mdx

```diff
@@ -0,0 +1,126 @@
+import { Meta, Controls, ArgTypes, Title, Story, Canvas, Description, Markdown } from '@storybook/blocks';
+import { e2eInteractionTest } from '../tests/index.stories';
+import * as ComponentStories from './ComponentStories.stories';
+
+<Meta of={ComponentStories} />
+
+<Title>Cio Quiz Component</Title>
+This component `<CioQuiz />` handles state management, data fetching, and rendering logic
+```jsx
+import CioQuiz from '@constructor-io/constructorio-ui-quizzes';
+
+function YourComponent() {
+  return (
+    <div>
+      <CioQuiz quizId='coffee-quiz' apiKey='key_wJSdZSiesX5hiVLt' />
+    </div>
+  );
+}
+``` 
+
+##### Usage Code Examples
+- [Basic Usage](#basic-usage)
+- [Provide CIO Client Instance](#provide-cio-client-instance)
+- [Handle Favorites On Results Page](#handle-favorites-on-results-page)
+
+<h2>Quiz Props</h2>
```

Comment:
I think a couple of the fields at the top do have defaults right?
* `enableHydration` - defaults to `false`?
* `primaryColor` - seems like this might have a default of a blue color? I could be wrong